fix(exporters): added save output button on rendered output

kpatl1 opened the pull request:
**Summary:**
added a confirmation dialog that shows which cell's output will be saved and instructs users to click on the desired cell first before using "Save Output"


**Testing:**
Created new Sasnb file and tried to save output from two different cells. 
**TODOs:**

- [ ] Add any supporting documentation and (optionally) update [CHANGELOG.md](CHANGELOG.md)

Fixes https://github.com/sassoftware/vscode-sas-extension/issues/1648

scnwwu commented:
I tried it and it still covers the content.
<img width="289" height="151" alt="image" src="https://github.com/user-attachments/assets/6a5cdf4f-830a-4591-8f20-400064ced7ed" />
I know it only appears on hover but some users may feel annoying. The code cell toolbar flows beyond the top border of the cell which will not cover any content. Are you able to make it to the similar position? (I guess a negative `top` style).

And, where is the icon from? To me, it doesn't look like the save/export or related icon we are familiar with.

kpatl1 commented:
> I tried it and it still covers the content. <img alt="image" width="289" height="151" src="https://private-user-images.githubusercontent.com/96556502/503052770-6a5cdf4f-830a-4591-8f20-400064ced7ed.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjEwOTg2OTcsIm5iZiI6MTc2MTA5ODM5NywicGF0aCI6Ii85NjU1NjUwMi81MDMwNTI3NzAtNmE1Y2RmNGYtODMwYS00NTkxLThmMjAtNDAwMDY0Y2VkN2VkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTEwMjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUxMDIyVDAxNTk1N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZjNmI0ZDk4MWRlOTgwMzBjOTM0ZDIyNTk5MWZhNzFiOWU4ZWI2YmJlMzNmMjU4ZjIyNDk3ZjlhMjllODQyNDQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.snxdRZRB3Tj6teezR2Le4yhX8i1YXM62cLhIUbiYzJg"> I know it only appears on hover but some users may feel annoying. The code cell toolbar flows beyond the top border of the cell which will not cover any content. Are you able to make it to the similar position? (I guess a negative `top` style).
> 
> And, where is the icon from? To me, it doesn't look like the save/export or related icon we are familiar with.

I added in a negative top, let me know what you think. I was also thinking of maybe adding it to the code cell toolbar instead what are your thoughts on that? I have also adjusted the icon to get one from the VSCode library.

kpatl1 commented:
Gotcha, I have gone and fixed the log output negative top as well.

scnwwu commented on client/src/components/notebook/renderers/LogRenderer.ts:
```diff
@@ -4,19 +4,93 @@ import type { ActivationFunction } from "vscode-notebook-renderer";
+    container.style.position = "relative";
+
+    if (context.postMessage) {
+      const toolbar = document.createElement("div");
```
are you able to re-use some code with HTMLRenderer?
